README.md

```diff
@@ -213,12 +213,19 @@ color:
   # When "classic" is set, this is set to "never".
   # Possible values: never, auto, always
   when: auto
+  # How to colorize the output.
+  # When "classic" is set, this is set to "no-color".
+  # Possible values: default, <theme-file-name>
+  # when specifying <theme-file-name>, lsd will look up theme file
+  # XDG Base Directory if relative, e.g. ~/.config/lsd/themes/<theme-file-name>,
```

Comment:
I think it would be better to automatically add the `.yaml` extension
So, it will be `~/.config/lsd/themes/<theme-file-name>.yaml`